pyodata/v2/model.py

```diff
@@ -28,6 +30,113 @@ def modlog():
     return logging.getLogger(LOGGER_NAME)
 
 
+class NullAssociation:
+    def __init__(self, name):
+        self.name = name
+
+    def __getattr__(self, item):
+        raise PyODataModelError('Cannot access this association. An error occurred during parsing '
+                                'association metadata due to that annotation has been omitted.')
+
+
+class NullType:
+    def __init__(self, name):
+        self.name = name
+
+    def __getattr__(self, item):
+        raise PyODataModelError(f'Cannot access this type. An error occurred during parsing '
+                                f'type stated in xml({self.name}) was not found, therefore it has been replaced with NullType.')
+
+
+class ErrorPolicy(ABC):
+    @abstractmethod
+    def resolve(self, ekseption):
+        pass
+
+
+class PolicyFatal(ErrorPolicy):
+    def resolve(self, ekseption):
+        raise ekseption
+
+
+class PolicyWarning(ErrorPolicy):
+    def __init__(self):
+        self._logger = logging.getLogger('pyodata')
+
+    def resolve(self, ekseption):
+        self._logger.warning(repr(ekseption))
+
+
+class PolicySilent(ErrorPolicy):
+    def resolve(self, ekseption):
+        pass
+
+
+class AutoNumber(enum.Enum):
```

Comment:
https://docs.python.org/3/library/enum.html#using-automatic-values

Reply:
Haven't found that before. Cool, I will rewrite to it.